docs/cookbook/customizing_form_builder_email_notifications.md

```diff
@@ -0,0 +1,24 @@
+# Customizing Form Builder email notifications
+
+!!! enterprise
+    
+    ## Override email template  
+    
+    Email is one of the Submit button options you can choose from in the Form Builder Interface.
+    It allows you to gather readers' email addresses to send notifications to.
```

Comment:
I still think it sounds a bit unclear. Now it looks like we're sendind an email to the people who fill in the form (that's how I would understand "readers"), while the email in fact goes to the editor(s) who manage the form. Can you confirm I'm correct, @adamwojs ?